Source/UnitTests/RingBufferTest.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2014 Dolphin Emulator Project
```

Comment:
Can you put that in the Common/ directory like other unit tests for Common/ libraries?